### PR TITLE
fix: wrong service account name

### PR DESCRIPTION
**Pull request by Vampouille**

### CODE REVIEW

**Comment by Vampouille** [comment]
CLA is signed

**Comment by T-Lakshmi** [comment]
Hi @Vampouille,
please sign EasyCLA, which is mandatory check to review your PR and accept the contribution.

You can find about EasyCLA here - https://docs.linuxfoundation.org/lfx/easycla

and the steps to sign EasyCLA are here - https://docs.linuxfoundation.org/v2/easycla/contributors/individual-contributor

You can start your signing EasyCLA from here - https://api.easycla.lfx.linuxfoundation.org/v2/repository-provider/github/sign/18706487/51478266/38056/#/?version=2

File: content/en/docs/tasks/configure-pod-container/configure-service-account.md [hunk 1]
```diff
@@ -174,7 +174,7 @@ Suppose you have an existing service account named "build-robot" as mentioned ea
 You can get a time-limited API token for that ServiceAccount using `kubectl`:
 
 ```shell
-kubectl create token admin-user
+kubectl create token build-robot
```
Comment (mengjiao-liu) [inline content/en/docs/tasks/configure-pod-container/configure-service-account.md:177]: PR https://github.com/kubernetes/website/pull/38321  has fixed it.

**Comment by mengjiao-liu** [comment]
This issue has been resolved in the latest documentation and you can now close the PR.
https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#manually-create-an-api-token-for-a-serviceaccount

**Comment by tengqm** [comment]
/close
Feel free to reopen if the issue is not resolved yet.

**Pull request closed without merging**